packages/reassure-cli/src/commands/measure.ts

```diff
@@ -110,6 +114,12 @@ export const command: CommandModule<{}, MeasureOptions> = {
       .option('commit-hash', {
         type: 'string',
         describe: 'Commit hash of current code to be included in the report',
+      })
+      .option('testMatch', {
+        alias: 'tm',
```

Comment:
It seems that Jest does not have an alias for `--testMatch`, if so let be compatible with it.
```suggestion
```